lib/dialects/postgres/data-types.js

```diff
@@ -123,6 +155,27 @@ module.exports = BaseTypes => {
     return 'BOOLEAN';
   };
 
+  BOOLEAN.prototype._sanitize = function _sanitize(value) {
```

Comment:
Why boolean is needed here, its logic looks exactly same as base type

Reply:
Where value === 'f' or 't'.

Reply:
👍